File: 1/344 김주완.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        len2 = len(s)
        for i in range(len2//2):
            tmp = s[i]
            s[i] = s[len2 - 1 - i]
            s[len2 - 1 - i] = tmp

File: 1/344 김주완.java
class Solution {
    public void reverseString(char[] s) {
        char tmp;
        int len = s.length;
        for (int i=0; i<len/2; i++) {
            tmp = s[i];
            s[i] = s[len - 1 - i];
            s[len - 1 - i] = tmp;
        }
    }
}
